src/api-v1.js

```diff
@@ -10,6 +10,7 @@ let GENERIC_ID_PATTERN = /^[a-zA-Z0-9-_]{1,22}$/;
 let api = new API({
   title: 'Cloud Mirror API',
   description: 'Service to duplicate URLs from various cloud providers',
+  name: 'cloudmirror',
```

Comment:
This should be `cloud-mirror`.  The `-` is allowed as long as it's not the first charcter:
https://github.com/taskcluster/taskcluster-lib-api/blob/master/src/api.js#L650